src/main/kotlin/no/nav/syfo/personhendelse/PersonhendelseService.kt

```diff
@@ -0,0 +1,56 @@
+package no.nav.syfo.personhendelse
+
+import no.nav.person.pdl.leesah.Endringstype
+import no.nav.person.pdl.leesah.Personhendelse
+import no.nav.syfo.application.database.DatabaseInterface
+import no.nav.syfo.domain.PersonIdentNumber
+import no.nav.syfo.oppfolgingstilfelle.OppfolgingstilfelleService
+import no.nav.syfo.personhendelse.db.addPerson
+import no.nav.syfo.personhendelse.db.getDodsdato
+import no.nav.syfo.util.kafkaCallId
+import org.slf4j.LoggerFactory
+
+class PersonhendelseService(
+    private val database: DatabaseInterface,
+    private val oppfolgingstilfelleService: OppfolgingstilfelleService,
+) {
+    suspend fun handlePersonhendelse(personhendelse: Personhendelse) {
+        if (personhendelse.doedsfall != null && personhendelse.endringstype == Endringstype.OPPRETTET) {
+            log.info("Received personhendelse: Hendelseid: ${personhendelse.hendelseId}, Endringstype: ${personhendelse.endringstype.name}, Oppltype: ${personhendelse.opplysningstype}, Dato: ${personhendelse.doedsfall?.doedsdato}")
+            personhendelse.personidenter
+                .mapNotNull { personIdent ->
+                    try {
+                        if (personIdent.isNullOrEmpty() || personIdent.length != 11) {
+                            null
+                        } else {
+                            PersonIdentNumber(personIdent)
+                        }
+                    } catch (ex: IllegalArgumentException) {
+                        log.warn("Invalid personident for Personhendelse", ex)
+                        null
+                    }
+                }
+                .forEach { personIdent ->
+                    if (isKnownPersonIdent(personIdent)) {
+                        database.connection.use { connection ->
+                            if (connection.getDodsdato(personIdent) == null) {
+                                connection.addPerson(
```

Comment:
Kan vurdere å endre til `.createPerson` - tror det er det vi stort sett bruker ellers. Og kunne man brukt samme connection for hver ident her eller spiller det ingen rolle?